### PR TITLE
Add .gitignore, .travis.yml, LICENSE and README.md files

### PR DESCRIPTION
**Pull request by jgrandguillaume**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+[![Build Status](https://travis-ci.org/OCA/rma.svg?branch=7.0)](https://travis-ci.org/OCA/rma)
+[![Coverage Status](https://img.shields.io/coveralls/OCA/rma.svg)](https://coveralls.io/r/OCA/rma?branch=7.0)
```
Comment (bwrsandman) [inline README.md:2]: You're better off using the png badge, so you can show the status for a specific branch

```
[![Coverage Status](https://coveralls.io/repos/OCA/rma/badge.png?branch=7.0)](https://coveralls.io/r/OCA/rma?branch=7.0)
```

Comment (pedrobaeza) [inline README.md:2]: SVG is not supported for an specific branch?

Comment (bwrsandman) [inline README.md:2]: As far as I know, there is no way to specify a branch using the svg badge for coverage

Comment (pedrobaeza) [inline README.md:2]: OK, then better png...

**Pull request merged**